Sources/StreamChatUI/CommonViews/ContainerView.swift

```diff
@@ -28,11 +28,11 @@ extension UILayoutPriority {
 }
 
 extension CGFloat {
-    static let auto: CGFloat = .infinity
+    public static let auto: CGFloat = .infinity
 }
 
 public class ContainerView: UIView {
-    convenience init(
+    public convenience init(
```

Comment:
hmm, this class is not documented at all, although it is public ... is it planned to document this later ... ?

Reply:
@fortmarek yes, still a bit WIP